src/forms/FormFieldAdvancedSelection/AdvancedSelectionDrawer.tsx

```diff
@@ -77,19 +103,8 @@ const AdvancedSelectionDrawer = (props: AdvanceSelectionDrawerPropTypes): ReactE
 		if (value?.length > 0 && isModalOpen) {
 			disableCheckboxList(value);
 			if (isMounted) {
-				dispatch(
-					formActions.setFieldValue({
-						name: "checkboxList",
-						value: value
-					})
-				);
-
-				dispatch(
-					formActions.setFieldValue({
-						name: "listOfChips",
-						value: value
-					})
-				);
+				dispatchChipList(value);
+				dispatchCheckboxList(value);

```

Comment:
We don't want to dispatchCheckboxList here because we already have a useEffect in place that dispatches the checkbox list every time the chipList changes. That way we ensure the "master" of the data is chipList.